test.js

```diff
@@ -119,6 +119,7 @@ describe('gulp-svgstore unit test', function () {
       var result = file.contents.toString()
       var target =
       '<svg xmlns="http://www.w3.org/2000/svg">' +
+      '<defs/>' +
```

Comment:
probably , well the grunt one doesnt do , i guessed doing the same was good enough !